src/components/App/App.js

```diff
@@ -5,42 +5,25 @@ import './App.css'
 
 class App extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
       movies: [],
-      currentMovieID: null,
-      currentMovie: null,
+      currentMovie: null
     }
   }
 
   handleClick = (event) => {
-    event.preventDefault();
-    this.setState({ currentMovieID: Number(event.target.id) })
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${Number(event.target.id)}`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ currentMovie: data.movie })
+      })
   }
 
   returnHome = (event) => {
-    event.preventDefault();
-    this.setState({ currentMovieID: null })
     this.setState({ currentMovie: null })
   }
 
-  componentDidUpdate() {
```

Comment:
Happy we found a solution and got out of the infinite loop we created in this method!